js/adapt-contrib-bookmarking.js

```diff
@@ -6,6 +6,7 @@ class Bookmarking extends Backbone.Controller {
     this.bookmarkLevel = null;
     this.restoredLocationID = null;
     this.currentLocationID = null;
+    this.currentContentObjectId = null;
```

Comment:
Could use `Adapt.location._currentId` as that holds the same data?

Reply:
Genius